### PR TITLE
fix: make python udf test run in ci

### PR DESCRIPTION
**Pull request by xudong963**

### CODE REVIEW

**Comment by chubei** [comment]
Does it work as is? You can check by `docker compose -f ./buildkite/build_dozer_tests/docker-compose.yaml up`

File: .github/workflows/coverage.yaml [hunk 1]
```diff
@@ -66,7 +66,7 @@ jobs:
         shell: bash
         run: |
           source ./dozer-tests/python_udf/virtualenv.sh
-          cargo test --all-features --no-fail-fast
+          cargo test --all-features --no-fail-fast --features python
```
Comment (v3g42) [inline .github/workflows/coverage.yaml:69]: does it not work with `all-features` ? do you need to explicitly include python ? Can you check. ?

Comment (xudong963) [reply .github/workflows/coverage.yaml:69]: My mistake, I didn't notice `all-features` 🤦

**Comment by xudong963** [comment]
> Does it work as is? You can check by `docker compose -f ./buildkite/build_dozer_tests/docker-compose.yaml up`

Yeah
```
725840fa9e34_build_dozer_tests-dozer-tests-1  |    Compiling tracing-actix-web v0.7.2
725840fa9e34_build_dozer_tests-dozer-tests-1  |    Compiling datafusion-expr v18.0.0
725840fa9e34_build_dozer_tests-dozer-tests-1  |    Compiling datafusion-row v18.0.0
725840fa9e34_build_dozer_tests-dozer-tests-1  |    Compiling actix-web-httpauth v0.8.0
725840fa9e34_build_dozer_tests-dozer-tests-1  |    Compiling datafusion-physical-expr v18.0.0
725840fa9e34_build_dozer_tests-dozer-tests-1  |    Compiling datafusion-sql v18.0.0
725840fa9e34_build_dozer_tests-dozer-tests-1  |    Compiling datafusion-optimizer v18.0.0
725840fa9e34_build_dozer_tests-dozer-tests-1  |    Compiling datafusion v18.0.0
725840fa9e34_build_dozer_tests-dozer-tests-1  |    Compiling dozer-orchestrator v0.1.7 (/Users/xudong/github/dozer/dozer-orchestrator)
725840fa9e34_build_dozer_tests-dozer-tests-1  |    Compiling dozer-tests v0.1.7 (/Users/xudong/github/dozer/dozer-tests)


725840fa9e34_build_dozer_tests-dozer-tests-1  |     Finished dev [unoptimized + debuginfo] target(s) in 19m 09s
```

**Comment by xudong963** [comment]
Close the ticket, it is involved by https://github.com/getdozer/dozer/pull/984/files#diff-28ede1b85c2e9b4b7d9ee401a73d27fa63d7e22bd855a8c57be1afdb675c6f93

**Pull request closed without merging**